File: LeetCode/1480/main.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& arr) {
        int n = arr.size();
        vector<int> ans;
        ans[0] = arr[0];
        for(int i = 1 ; i < n; i++) {
            ans[i] = ans[i-1] + arr[i];
        }
        return ans;
    }
}; 

File: LeetCode/1480/main.py
"""
DIFFICULTY : easy
TAGS : array, prefix sum
"""


class Solution:
    def runningSum(self, arr: List[int]) -> List[int]:
        pre = [None] * len(arr)
        pre[0] = arr[0]
        for i in range(1, len(arr)):
            pre[i] = pre[i - 1] + arr[i]
        return pre
